ckanext/harvest/utils.py

```diff
@@ -216,18 +216,12 @@ def clear_harvest_source_history(source_id, keep_current):
     if source_id is not None:
         tk.get_action("harvest_source_job_history_clear")(context, {
             "id": source_id,
-            "keep_current": keep_current
-        })
+            })
         return "Cleared job history of harvest source: {0}".format(source_id)
```

Comment:
Note the language used here.   This command is most useful if it clears the "job history", not the entire source history.    Perhaps a change in the command name to "clear-job-history" would be clearer.